File: Android/Konquest/app/src/main/java/ca/qc/cstj/konquest/models/Runes.kt
package ca.qc.cstj.konquest.models

import com.github.kittinunf.fuel.android.core.Json

/**
 * Created by trist on 18/12/2017.
 */
class Runes(jsonObject: Json) {
    var air : String = jsonObject.obj().getString("air")
    var darkness : String = jsonObject.obj().getString("darkness")
    var earth : String = jsonObject.obj().getString("earth")
    var energy : String = jsonObject.obj().getString("energy")
    var fire : String = jsonObject.obj().getString("fire")
    var life : String = jsonObject.obj().getString("life")
    var light : String = jsonObject.obj().getString("light")
    var logic : String = jsonObject.obj().getString("logic")
    var music : String = jsonObject.obj().getString("music")
    var space : String = jsonObject.obj().getString("space")
    var toxic : String = jsonObject.obj().getString("toxic")
    var water : String = jsonObject.obj().getString("water")
}

File: ajax-bootstrap4/ajax-bootstrap4/js/runes.js
const URLRUNES="https://andromia-maxroy1993.c9users.io/runes";


$(document).ready(function(){
   
   
    $.ajax({
           type: "GET",
           url: URLRUNES,
		   headers: {"Authorization": "bearer " + sessionStorage.getItem("Authorization") },
           success: function(runes)
           {
			   
               $("#runeAirQte").append(runes["air"]);
			   $("#runeDarknessQte").append(runes["darkness"]);
			   $("#runeEarthQte").append(runes["earth"]);
			   $("#runeEnergyQte").append(runes["energy"]);
			   $("#runeFireQte").append(runes["fire"]);
			   $("#runeLifeQte").append(runes["life"]);
			   $("#runeLightQte").append(runes["light"]);
			   $("#runeLogicQte").append(runes["logic"]);
			   $("#runeMusicQte").append(runes["music"]);
			   $("#runeSpaceQte").append(runes["space"]);
			   $("#runeToxicQte").append(runes["toxic"]);
			   $("#runeWaterQte").append(runes["water"]);
           },
		   error: function(error)
		   {
			    //window.location.href="./error.html";
		   }
         });
   
    
    
});